docs/contributing/CONTRIBUTING.md

```diff
@@ -0,0 +1,131 @@
+# Contributing Guidelines
+
+*We're so excited that you're interested in contributing to Neurobagel!* :partying_face:
+
+We appreciate all contributions, and hope that the below guidelines will make it as easy as possible for you to contribute to the [Neurobagel codebase](https://github.com/neurobagel) and to ensure your contribution can be easily integrated.
+
+## Identifying an issue to work on
+
+The easiest way to get started contributing is to explore the list of open issues in one of our [GitHub repositories](https://github.com/orgs/neurobagel/repositories). 
+For example, see the [open issues for the Neurobagel API](https://github.com/neurobagel/api/issues).
+
+Regardless of if you would like to contribute a feature, bug fix, or another type of change, we recommend that you first either
```

Comment:
```suggestion
When you are ready to contribute, the best way to join the conversation is by opening an issue to describe a change you would like to see or would like to contribute. Opening an issue before you start contributing code is important because it allows others in the project to discuss your idea and tell you where your contribution would be the most helpful.```

Something like that?